### ISSUE TITLE
Ratified Version Document

### ISSUE THREAD

**Issue by henry-hsieh** [riscv/riscv-count-overflow#6]
Sscofpmf is ratified in 2021. However, the version of document is 0.5.2-frozen. Does the ratification bump the version to 1.0-ratified?

**Comment by Passer-Andes** [comment]
I saw a document with the tag v1.0.0, but the cover page of the source code still shows the version is 0.5.2-frozen.

**Comment by pdonahue-ventana** [comment]
@jjscheel: Is there anything that can be done about this?  This extension is mandatory in RVA23 so the spec shouldn't be in this semi-abandoned pre-ratification state.

**Comment by jjscheel** [comment]
This information will be included in the main ISA manual soon and all references to the previous doc will be archived.  So, an update for a version/title page seems not worth the time.  Would you agree?

**Comment by jjscheel** [comment]
@wmat, can you confirm that Sscofpmf  is in your Asciidoc chapters which will be built for the ISA manuals we're building?  If so, would you kindly provide a link to the branch here so that we can close this out with complete information?

**Comment by wmat** [comment]
@jjscheel it will be in priv (after Svinval) but is not there yet.

**Comment by pdonahue-ventana** [comment]
That's fine if it is actually soon.  One of the checklist items post-ratification should be to update things like this.  That obviously wasn't the case in 2021 when this was ratified.  This same problem also affects [Sstc](https://github.com/riscv/riscv-time-compare/releases).

**Comment by jjscheel** [comment]
Good point, @pdonahue-ventana.

@rpsene, perhaps we can get a few sub-tasks in Jira for the following?

1. Updating the document to reflect "Ratified" state and building PDF.
2. Creating and submitting a final PR to riscv-isa-manual (for ISA and Fast Track only).
3. Publishing the Ratified document.

**Comment by rpsene** [comment]
<img width="1008" alt="Screenshot 2024-02-01 at 09 59 15" src="https://github.com/riscv/riscv-count-overflow/assets/20800562/ba69cfed-0a0d-4a40-b553-da0fa0676e5b">

@jjscheel we already have it; we can improve the description of the item. But we can also add a post-ratification task with this items.

**Comment by jjscheel** [comment]
No, that is the review item to make sure the repo is in a good state.  It occurs BEFORE ratification.

@pdonahue-ventana, is talking about actions which must occur post ratification.  They aren't really "Ecosystem Development", but rather completion of the Ratification work.
